etc/config/fsharp.defaults.properties

```diff
@@ -0,0 +1,17 @@
+compilers=fsharp
+supportsBinary=false
+needsMulti=false
+compilerType=fsharp
+
+compiler.fsharp.exe=/opt/compiler-explorer/dotnet-6.0/.dotnet/dotnet
+compiler.fsharp.name=.NET 6 (x64)
+
+compiler.fsharp.clrDir=/opt/compiler-explorer/dotnet-6.0/artifacts/bin/coreclr/Linux.x64.Checked
+compiler.fsharp.runtimeId=linux-x64
+compiler.fsharp.targetOS=linux
+compiler.fsharp.targetArch=x64
+compiler.fsharp.instructionSet=base,sse,sse2,sse3,ssse3,sse4.1,sse4.2,avx,avx2,aes,bmi,bmi2,fma,lzcnt,pclmul,popcnt,avxvnni
```

Comment:
This is not how the (already existing) instructionSet property works. You probably want to rename this one to something like supportedInstructionSets and use that over here https://github.com/compiler-explorer/compiler-explorer/pull/3168/files#diff-e84770ce2518a6f397e8c06c65be9979d042091415ecd20b6a133159bd2bff37R140

Reply:
Done.